poetry/core/masonry/builders/wheel.py

```diff
@@ -168,9 +168,20 @@ def _write_metadata(self, wheel):
             with self._write_to_zip(wheel, self.dist_info + "/entry_points.txt") as f:
                 self._write_entry_points(f)
 
+        license_files_to_add = []
         for base in ("COPYING", "LICENSE"):
-            for path in sorted(self._path.glob(base + "*")):
-                self._add_file(wheel, path, "%s/%s" % (self.dist_info, path.name))
+            license_files_to_add.append(self._path / base)
+            license_files_to_add.extend(self._path.glob(base + ".*"))
```

Comment:
```suggestion
            license_files_to_add.append(base)
            license_files_to_add.extend(self._path.glob(base + ".*"))
```

Reply:
This line (and the `path.relative_to(...)` line later) are necessary because the license files must be found relative to `self._path`, and because otherwise, when we add the files to the zip file, they'll come out containing absolute parts (e.g. `project/tmp/testdir-7843923/...`).

We could probably just use a string here (that was my first attempt), but juggling a mixed array of strings _and_ `Path` objects made the code more complicated, whereas converting everything into an absolute `Path` object relative to the `self._path` variable means that when we later add the files to the zip, we need to do a lot less work to make sure they're all in the right format.

Reply:
Hmm, that is fine. I just wanted to avoid the possibility of ending up with a ValueError. But considering we control what goes into the list, this should be fine. No strong feelings on this one :).

Reply:
Yeah, I tried `Path(base)` briefly, but I think it's helpful to have either all the paths as absolute, or none of them.  But I think it works reasonably like this, and the fact that tests fail when I go for different options gives me more confidence that this is a fairly stable approach that shouldn't run into `ValueError` problems.